src/group/managed_group/mod.rs

```diff
@@ -750,6 +797,13 @@ impl<'a> ManagedGroup<'a> {
         }
     }
 
+    /// Auto-save function
+    fn auto_save(&self) {
+        if let Some(auto_save) = self.managed_group_config.callbacks.auto_save {
+            auto_save(&self);
```

Comment:
We should test this.

Reply:
I added that at the end of the integration tests.